dds_web/api/schemas/project_schemas.py

```diff
@@ -135,8 +135,10 @@ def create_project(self, data, **kwargs):
                 public_id=data["public_id"], created_time=data["date_created"]
             )
 
-            # Generate keys
+            # NOTE: TEMPORARY
+            # Generate keys and add to project
             data.update(**key_gen.ProjectKeys(data["public_id"]).key_dict())
```

Comment:
This is old key_gen code that generates a public and private key pair and its product is saved into the project model